docs/plasma/index.rst

```diff
@@ -74,23 +75,30 @@ which the data match its plasma data structure.
 Plasma Subclasses
 -----------------
 
-The :class:`Plasma3D` class is a basic structure to contain spatial
-information about a plasma.  To initialize a Plasma3D system, first
-create an instance of the :class:`Plasma3D` class and then set the
-:attr:`Plasma3D.density`, :attr:`Plasma3D.momentum`,
-:attr:`Plasma3D.pressure` and the :attr:`Plasma3D.magnetic_field`.
+The :class:`~plasmapy.plasma.sources.plasma3d.Plasma3D` class is a basic
+structure to contain spatial information about a plasma.  To initialize
+a Plasma3D system, first create an instance of the
+:class:`~plasmapy.plasma.sources.plasma3d.Plasma3D` class and then set the
+:attr:`~plasmapy.plasma.sources.plasma3d.Plasma3D.density`,
+:attr:`~plasmapy.plasma.sources.plasma3d.Plasma3D.momentum`,
+:attr:`~plasmapy.plasma.sources.plasma3d.Plasma3D.pressure` and
+attr:`~plasmapy.plasma.sources.plasma3d.Plasma3D.magnetic_field`.
```

Comment:
I wonder whether we might not be able to save on the line count by moving this whole doc into the beginning docstring of `plasmapy/plasma/<...somewhere...>.py`. That'd also help us keep the documentation closer to the code it describes.

Reply:
Just created #1517 since this woudl probably warrant its own PR.